http.go

```diff
@@ -61,8 +61,9 @@ func (c *Client) Request(method string, path string, body []byte) (*http.Respons
 
 	apiVersion := res.Header.Get("DEIS_API_VERSION")
 
-	// Update controller api version
+	// Update controller api and platform version
 	c.ControllerAPIVersion = apiVersion
+	c.DeisVersion = res.Header.Get("DEIS_PLATFORM_VERSION")
```

Comment:
I think it's probably unnecessary. If it's empty, the version object will remain empty (as default). It's not easy to check if the version if malformed. It's a string, so there isn't really any consequence if it is malformed. All the cli is going to do with it is log the value.